### DISCUSSION TITLE
Code location?

### DISCUSSION THREAD

**Discussion by Grandma-Betty** [KostyaEsmukov/smtp_to_telegram#35] [Q&A] [answered]
I mentioned that docker images are still being update regularly, the last one was updated 4 days ago.
So why isn't the code on GitHub being updated? Where are you actually hosting your current code?
Thanks in advance for being transparent!

**Comment by KostyaEsmukov** [comment, marked as answer]
The code is located in this exact repo.

The image is automatically rebuilt weekly from the current master of this repo:
```
$ docker run --rm kostyaesmukov/smtp_to_telegram --version
smtp_to_telegram version 0.1-37-g45b6594
```
Where 45b6594 corresponds to the current master.

The reason for the weekly rebuilds is to keep the dependencies (mostly from alpine) up to date.

**Reply by Grandma-Betty** [reply]
Fair enough mate, thank you for clearing things up!
